k8s/regions/frankfurt/test.yaml

```diff
@@ -0,0 +1,147 @@
+kubernetes:
+    app_name: "sumo-dev"
+    namespace: "sumo-test"
+    target_environment: "test"
+    nodeport_name: "sumo-nodeport"
+    secrets_name: "sumo-secrets-dev"
+    image:
+        repo: "itsre/sumo-kitsune"
+        tag: "full-latest"
+        pull_policy: "Always"
+    # default values
+    limits:
+        memory_request: 384Mi
+        memory_limit: 1Gi
+        cpu_request: 250m
+        cpu_limit: 1000m
+    replicas: 1
+    rollouts:
+        max_surge: 25%
+        max_unavailable: 25%
+    readiness: False
+    liveness: False
+    # app specific values
+    apps:
+        cron:
+            command: "./bin/run-cron.sh"
+            deployment_name: "sumo-dev-cron"
+            labels:
+                type: "cron"
+        celery:
+            command: "./bin/run-celery-worker.sh"
+            deployment_name: "sumo-dev-celery"
+            replicas: 1
+            labels:
+                type: "celery"
+        web:
+            command: "./bin/run-prod.sh"
+            deployment_name: "sumo-dev-web"
```

Comment:
we'll want to sub sumo-test basically anywhere we sumo-dev. Are we going to have cron/celery as part of this environment deployment?

Reply:
I would say so. I am updating all the references